src/.idea/.idea.MockBot/.idea/.gitignore

```diff
@@ -0,0 +1,17 @@
+# Default ignored files
```

Comment:
What is the "idea" ( :) ) of this file, it looks like a strange path?

Reply:
.idea is where IntelliJ/Rider keeps its configs, I must have mangled the import terribly. Ideally I would ignore this entire folder, will try to delete it from the PR.